src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -163,11 +163,25 @@ export const CheckoutPayment = ({
   const [getStatus] = useCheckoutStatusLazyQuery({
     pollInterval: 1000,
   })
-  const { search: is3DsComplete } = useLocation<{ search: string }>()
+  const history = useHistory()
```

Comment:
Why not still use `useLocation`?

Reply:
I guess he needs `history` so he can replace `error` search param below.

Reply:
Aha I see, but I meant for accessing search params. That seems a bit more complicated now, no?

```
const {
    location: { search },
  } = history
```

But just a nit

Reply:
yeah, I think it's better performance to destructure that object rather than using 2 hooks